examples/rails/README.md

```diff
@@ -0,0 +1,15 @@
+# Profile Example - _Ruby on Rails_
+
+1. Create your application in the [Yoti Hub](https://hub.yoti.com)
+1. Set the application domain of your app to `localhost:3001`
+1. Set the scenario callback URL to `/profile`
+1. Rename the [.env.example](examples/rails/.env.example) file to `.env` 
+1. Fill in the environment variables in this file with the ones specific to your application (mentioned in the [Configuration](#configuration) section)
```

Comment:
Configuration link doesn't work now it's in a separate README. I just mentioned the Hub in .NET Core, but just realised I need to update the 4.7 Example one
```suggestion
1. Fill in the environment variables in this file with the ones specific to your application, generated in the Yoti Hub when you create (and then publish) your application
```